doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -0,0 +1,329 @@
+When all CPUs are ready to fully minimize power consumption by entering the System Off hardware state, the System Controller prepares the system and triggers the entrance in this state.
+
+Power states
+************
+
+The nRF54H20 ARM Cortex CPUs currently support the following software power states:
+
+* Active
+* Idle
+* Local suspend to Ram
+
+Overview
+********
+
+Each CPU in the nRF54H20 SoC tries to preserve as much power as possible, independently from other CPUs.
+The power management subsystem, operating independently within each CPU, continuously selects the most optimal power state based on the current conditions of the CPU.
+
+Power states details
+********************
+
+The following sections describes the details of each of the software power states available on the nRF54H20 SoC.
+
+Active
+======
+
+*Active* is the power state in which the CPU is actively processing.
+
+.. list-table::
+   :widths: auto
+
+   * - CPU
+     - Powered on and clocked.
+
+   * - RAM
+     - Banks used by the executed code are enabled.
+       Other banks may be in any state.
+
+   * - System state
+     - *System ON*
+
+   * - Peripherals
+     - All can be active.
+       The state of all inactive peripherals is retained in the hardware flip-flops.
+
+   * - Coprocessors
+     - All can be active.
+       The state of inactive coprocessors is retained according to their selected sleep state.
+
+   * - System timer (based on GRTC)
+     - Active
+
+This is the power state with the highest power consumption.
+The software execution latency is minimal.
+The primary source of latency is the wake-up of the ``MRAMC``.
+The maximum latency depends on the MRAM power state configured by the System Controller.
+
+Idle
+====
+
+*Idle* is the lightest sleep power state available in the system where the cache content is retained.
+
+.. list-table::
+   :widths: auto
+
+   * - CPU
+     - Powered on but suspended (not clocked).
+
+   * - RAM
+     - Banks used by the executed code are enabled.
+       Other banks may be in any state.
+
+   * - System state
+     - *System ON*
+
+   * - Peripherals
+     - All can be active.
+       The local peripherals are powered and preserve their state.
+       The state of inactive peripherals in other domains is retained in the hardware flip-flops.
+
+   * - Coprocessors
+     - All can be active.
+       The state of inactive coprocessors is retained according to their selected sleep state.
+
+   * - System timer (based on GRTC)
+     - Active
+
+The power consumption in this state is lower than in *Active*, but higher than in other power states.
+The wake-up latency is higher than in *Active*, but lower than in the other power states.
+The primary sources of wake-up latency are the wake-up of the ``MRAMC`` and the startup of clock sources.
+The maximum latency depends on the MRAM power state configured by the System Controller.
+
+Idle with cache retained in RAM
```

Comment:
Now that we don't list to-be-implemented states, I think we should remove them from this section